Softeq.XToolkit.Common/Timers/Timer.cs

```diff
@@ -13,20 +13,27 @@ namespace Softeq.XToolkit.Common.Timers
     /// </summary>
     public class Timer : ITimer, IDisposable
     {
-        private readonly int _interval;
-        private TaskReference? _taskReference;
-
         /// <summary>
         ///     Initializes a new instance of the <see cref="Timer"/> class with specified interval.
         /// </summary>
         /// <param name="taskReference">Task to be executed at specified interval.</param>
         /// <param name="interval">Timer interval (ms).</param>
         public Timer(TaskReference taskReference, int interval)
         {
-            _taskReference = taskReference;
-            _interval = interval;
+            TaskReference = taskReference;
+            Interval = interval;
         }
 
+        /// <summary>
+        /// Gets an interval setting of this timer.
+        /// </summary>
+        public int Interval { get; }
```

Comment:
I think no need to expose these properties, it's out of the scope of current class